controllers/MoveController.php

```diff
@@ -63,4 +65,76 @@ public function actionIndex() //Make sure an $fid is given otherwise the root fo
             return $this->htmlRedirect($model->sourceFolder->createUrl('/cfiles/browse'));
         }
     }
+
+    public function actionDrop()
+    {
+        $this->forcePostRequest();
+
+        $targetFolder = $this->getTargetFolder(Yii::$app->request->post('targetFolder'));
+        if (!$targetFolder) {
+            return $this->asJson([
+                'error' => Yii::t('CfilesModule.base', 'Wrong target folder!'),
+            ]);
+        }
+
```

Comment:
Is there some sort of `$droppedItem->canEdit()` required?

Reply:
@luke- Yes, you are right, I have to be more careful. Fixed in commit https://github.com/humhub/cfiles/pull/132/commits/8b633d144593f96649e27c62fa9916752bb8260d.

Reply:
@yurabakhtin Thanks!